orders/models.py

```diff
@@ -0,0 +1,76 @@
+from django.db import models
+from django.utils import timezone
+from daycare.models import DayCare
+from dogowner.models import DogOwner
+
+
+class StatusOptions(models.TextChoices):
+    Pending = 'P', 'Pending'
+    Approved = 'A', 'Approved'
+    Canceled = 'C', 'Canceled'
+    Finished = 'F', 'Finished'
+    OnGoing = 'O', 'On Going'
+
+
+class Order(models.Model):
+    order_id = models.AutoField(primary_key=True)
+    dog_owner_id = models.ForeignKey(DogOwner, on_delete=models.CASCADE)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.CASCADE)
```

Comment:
> I think that on_delete should be models.do_nothing. As DogOwner I wish to see all orders that I did even if the DayCare was deleted As DayCare I wish to see all orders that I had even if the DogOwner deleted.
> 
> We should delete the order only if both deleted

The "DO_NOTHING" will not work, because when a user (dog owner / daycare) will be deleted from the database, the FK will refer to a non-existed instance, and it's not possible.
I'll change it to "SET_NULL" instead.

Reply:
Note that this should be cover by tests.

Reply:
Check out `set` and `set_null` options.

Reply:
I think that for keeping it MVP, we should not support a situation when we keep the order when one of the users gets deleted.
Also, IMO, the idea of setting the foreign key to null has no logic in it.
If we do accept null as a proper value for the dog owner / daycare, we're saying that it's ok for the order to not have 2 sides of "the deal".

Reply:
You do not accept null usually, but only in cases like this when one user gets deleted. then, the view can show an "unavailable user" whenever it is null.

You can also create a default "unavailable user" to set it as, but I think this is not the right approach.